src/patternfly/components/Brand/brand.scss

```diff
@@ -1,4 +1,17 @@
-// stylelint-disable
+$pf-c-brand--breakpoint-map: build-breakpoint-map();
+
 .pf-c-brand {
+  --pf-c-brand--Width--base: auto;
+  --pf-c-brand--Height--base: auto;
+
+  width: var(--pf-c-brand--Width--base);
+  max-width: 100%;
```

Comment:
I'm wondering if we should also move this in to `&.pf-m-picture`. I'm trying to think of how this might break someone, and if they have applied `.pf-c-brand` to something other than an `img`, and had custom CSS loaded before patternfly, this might change their layout. An `img` should already get it from https://github.com/patternfly/patternfly/blob/51955caf5d1dba384a7cfa1dd454d20f1caf5d1c/src/patternfly/base/_globals.scss#L65-L73

WDYT about moving it in to `.pf-m-picture`, and leaving a comment to move it in the next breaking change release?

Reply:
WFM